### PR TITLE
Fix random function issue. DHH-847

### PR DESCRIPTION
**Pull request by VivianChu**

### CODE REVIEW

File: lib/dams/random_controller_helper.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 module Dams
   module RandomControllerHelper
     # Retrieve random page from solr index
-    def random_page(q)
-      @response, @document = get_search_results(:q => "#{q}", :rows => 1)
-      page_count = @response.response['numFound']/20.to_f   
+    def random_page(q, fq)
+      @response, @document = get_search_results(:qf => "#{q}", :fq => "#{fq}", :rows => 100)
```
Comment (mcritchlow) [inline lib/dams/random_controller_helper.rb]: overall 👍 

would it be faster to have a lower number of initial results returned, which I suppose would result in a larger number of pages to sample from? Do we need 100, or could we do something like 5 or 10?

Comment (VivianChu) [inline lib/dams/random_controller_helper.rb]: @mcritchlow You're right, I forgot to change it back to 1 when doing testing.  
For that function, we just need to get the number of results found.  I will change it back to 1.

**Comment by lsitu** [comment]
👍

**Comment by VivianChu** [comment]
Thanks everyone for reviewing.  Could anybody please merge this PR?

**Pull request merged**